content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,724 @@
+for the load balancer to register the ingress gateway.
+
+## Environment Variables
+
+This guide will refer to two clusters named `FRED` and `BARNEY`. The following
+environment variables will be used throughout to simplify the instructions:
+
+Variable | Description
+-------- | -----------
+`CTX_FRED` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the `FRED` cluster.
+`CTX_BARNEY` | The context name in the default [Kubernetes configuration file](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/) used for accessing the `BARNEY` cluster.
+
+For example:
+
+{{< text bash >}}
+$ export CTX_FRED=cluster-fred
+$ export CTX_BARNEY=cluster-barney
+{{< /text >}}
+
+## Configure Trust
+
+A multicluster service mesh deployment requires that you establish trust
+between all clusters in the mesh. Depending on the requirements for your
+system, there may be multiple options available for establishing trust.
+See [certificate management](/docs/tasks/security/cert-management/) for
+detailed descriptions and instructions for all available options.
+Depending on which option you choose, the installation instructions for
+Istio may change slightly.
+
+This guide will assume that you use a common root to generate intermediate
+certificates for each cluster. Follow the [instructions](/docs/tasks/security/cert-management/plugin-ca-cert/)
+to generate and push a secret to both the `FRED` and `BARNEY` clusters.
+
+{{< tip >}}
+If you currently have a single cluster with a self-signed CA (as described
+in [Getting Started](/docs/setup/getting-started/)), you’ll need to
+[change the CA](/docs/tasks/security/cert-management/) to support multiple
+clusters and then reinstall Istio using the standard installation
+instructions below.
+{{< /tip >}}
+
+## Install Istio
+
+The steps for installing Istio on multiple clusters depend on your
+requirements for network and control plane topology. This section
+illustrates the options for two clusters. Large systems may employ more
+than one of these options for meshes that span many clusters. See
+[deployment models](/docs/ops/deployment/deployment-models) for more
+information.
+
+{{< tabset category-name="install-istio" >}}
+
+{{< tab name="Multi-Primary" category-value="multi-primary" >}}
+
+The steps that follow will install the Istio control plane on both `FRED` and
+`BARNEY`, making each a {{< gloss >}}primary cluster{{< /gloss >}}. Both
+clusters reside on the `EAST` network.
+
+Each control plane observes the API Servers in both clusters for endpoints.
+Service workloads communicate directly (pod-to-pod) across cluster boundaries.
+
+{{< image width="75%"
+    link="multi-primary.svg"
+    caption="Multiple primary clusters on the same network"
+    >}}
+
+### Configure `FRED` as a primary
```

Comment:
All of the section headings that are inside a `{{< tab ... >}}` don't work properly. Notice that if you click on any of these headings in the outline, it doesn't go to them. Not sure, what the best solution is, but one option is just to turn these headings into bold text.
```suggestion
**Configure `FRED` as a primary**
```

Reply:
@frankbu if we make this change they simply won't appear in the outline, right? I'm not sure if that's much better. From a readability standpoint, having them as headings (rather than just bold) seems more clear.